Dockerfile-linux.template

```diff
@@ -573,13 +573,13 @@ RUN set -eux; \
 		echo '; Ensure worker stdout and stderr are sent to the main error log.'; \
 		echo 'catch_workers_output = yes'; \
 		echo 'decorate_workers_output = no'; \
+		echo; \
+		echo '; default listen adddress for easy override in later php-fpm.d/*.conf files'; \
+		echo 'listen = 9000'; \
 	} | tee php-fpm.d/docker.conf; \
 	{ \
 		echo '[global]'; \
 		echo 'daemonize = no'; \
-		echo; \
-		echo '[www]'; \
-		echo 'listen = 9000'; \
```

Comment:
This being in `zz-docker.conf` is a little bit load-bearing unless we want to modify files shipped with/by PHP itself: :sweat_smile:

```console
$ docker run --rm --pull=always php:fpm grep -rn '^listen' /usr/local/etc/php-fpm.d
fpm: Pulling from library/php
Digest: sha256:baa4dd163e5c9e95c4ba31773173b408c69c824f6a21da56af274a47e7ddbee6
Status: Image is up to date for php:fpm
/usr/local/etc/php-fpm.d/www.conf.default:41:listen = 127.0.0.1:9000
/usr/local/etc/php-fpm.d/www.conf:41:listen = 127.0.0.1:9000
/usr/local/etc/php-fpm.d/zz-docker.conf:5:listen = 9000
```